app/views/parser_versions/_form.html.erb

```diff
@@ -1,4 +1,5 @@
 <%= form_tag do %>
+  <%= hidden_field_tag "parser[id]", @parser.id %>
```

Comment:
Single quotes for `parser[id]`.

Reply:
I just took the same convention as the rest of that file. Looks like we're not very consistent on this across the project. I'll try to clean it up in another PR.